lib/components/app/call-taker-panel.js

```diff
@@ -158,15 +158,15 @@ class CallTakerPanel extends Component {
             {Array.isArray(intermediatePlaces) && intermediatePlaces.map((place, i) => {
               return (
                 <IntermediatePlace
-                  key={i}
                   index={i}
+                  inputPlaceholder={`Enter intermediate destination`}
+                  key={i}
                   location={place}
-                  onLocationCleared={this._removePlace}
                   // FIXME: function def
-                  onLocationSelected={result => this._addPlace(result, i)}
-                  // FIXME: allow intermediate location type.
                   locationType='to'
-                  inputPlaceholder={`Enter intermediate destination`}
+                  // FIXME: allow intermediate location type.
```

Comment:
Comment is now out-of-place.

Reply:
Good catch, thanks, fixed in https://github.com/opentripplanner/otp-react-redux/pull/390/commits/1a526dd2886f67b837dfe118666f9d9d719988f5.